packages/components/tabs/tabs.js

```diff
@@ -1,313 +1,303 @@
-// import "../../vendor/polyfills/Function/prototype/bind";
-// import "../../vendor/polyfills/Element/prototype/classList";
-// import "../../vendor/polyfills/Element/prototype/nextElementSibling";
-// import "../../vendor/polyfills/Element/prototype/previousElementSibling";
-// import "../../vendor/polyfills/Event"; // addEventListener and event.target normaliziation
-
-function Tabs($module, namespace, responsive, historyEnabled) {
-  this.$module = $module;
-  this.namespace = namespace;
-  this.responsive = responsive;
-  this.historyEnabled = historyEnabled;
-  this.$tabs = $module.querySelectorAll(`.${this.namespace}__tab`);
-
-  this.keys = {
-    down: 40, left: 37, right: 39, up: 38,
-  };
-  this.jsHiddenClass = `${this.namespace}__panel--hidden`;
-
-  this.showEvent = new CustomEvent('tab.show');
-  this.hideEvent = new CustomEvent('tab.hide');
-}
+class Tabs {
+  constructor($module, namespace, responsive, historyEnabled) {
+    this.$module = $module;
+    this.namespace = namespace;
+    this.responsive = responsive;
+    this.historyEnabled = historyEnabled;
+    this.$tabs = $module.querySelectorAll(`.${this.namespace}__tab`);
+
+    this.keys = {
+      down: 40, left: 37, right: 39, up: 38,
+    };
+    this.jsHiddenClass = `${this.namespace}__panel--hidden`;
+
+    this.showEvent = new CustomEvent('tab.show');
+    this.hideEvent = new CustomEvent('tab.hide');
+  }
 
-Tabs.prototype.init = function init() {
-  if (typeof window.matchMedia === 'function' && this.responsive) {
-    this.setupResponsiveChecks();
-  } else {
-    this.setup();
+  init() {
+    if (typeof window.matchMedia === 'function' && this.responsive) {
+      this.setupResponsiveChecks();
+    } else {
+      this.setup();
+    }
   }
-};
 
-Tabs.prototype.setupResponsiveChecks = function setupResponsiveChecks() {
-  this.mql = window.matchMedia('(min-width: 40.0625em)');
-  this.mql.addListener(this.checkMode.bind(this));
```

Comment:
IDE was showing addListener as deprecated

Reply:
yeah that's fair, good spot